docs/hub/spaces-gpus.md

```diff
@@ -39,7 +39,7 @@ In the following tables, you can see the Specs for the different upgrade options
 | Nvidia A10G - large    | 12 vCPU       | 46 GB        | 24 GB           | 200 GB    | $1.50             |
 | 2x Nvidia A10G - large | 24 vCPU       | 92 GB        | 48 GB           | 1000 GB   | $3.00             |
 | 4x Nvidia A10G - large | 48 vCPU       | 184 GB       | 96 GB           | 2000 GB   | $5.00             |
-| Nvidia A100 - large    | 12 vCPU       | 142 GB       | 40 GB           | 1000 GB   | $4.00             |
+| Nvidia A100 - large    | 12 vCPU       | 144 GB       | 80 GB           | 1000 GB   | $4.00             |
```

Comment:
Looking at https://github.com/huggingface-internal/workloads/pull/2535/files

It's still 142 GB RAM
```suggestion
| Nvidia A100 - large    | 12 vCPU       | 142 GB       | 80 GB           | 1000 GB   | $4.00             |
```